include/temoto_robot_manager/robot_features.h

```diff
@@ -262,6 +262,13 @@ class FeatureCustom : public FeatureWithDriver
   FeatureCustom(const std::string& name, const YAML::Node& yaml_node);
 };
 
+// Common Procedures Feature
+class FeatureCommon : public RobotFeature
```

Comment:
just like in my other comment, is there a benefit of using `RobotFeature` as a base class for common procedure?

Reply:
Renamed: FeatureCommon --> CommonProcedure
I think it is fine to use the RobotFeature as base class so I can inherit the methods on the CommonProcedure class (such as getName, getExecutable, ... ), instead of having to replicate them.